backend/modules/sse/serializers.py

```
@@ -1,8 +1,11 @@
+from typing import override
```

Pylint (3.3.7)

[error] No name 'override' in module 'typing'

(E0611)